vmm/src/vm.rs

```diff
@@ -1061,6 +1061,10 @@ impl Vm {
 
         Ok(())
     }
+
+    pub fn get_config(&self) -> VmConfig {
+        self.config.clone()
```

Comment:
Have you considered making it an `Arc` or an `Arc<Box>` ?

Reply:
`Vm` now takes an `Arc<VmConfig>`